src/node/main.ts

```diff
@@ -122,6 +123,17 @@ export const runCodeServer = async (
 ): Promise<{ dispose: Disposable["dispose"]; server: http.Server }> => {
   logger.info(`code-server ${version} ${commit}`)
 
+  // Load custom strings if provided
+  if (args["custom-strings"]) {
+    try {
+      await loadCustomStrings(args["custom-strings"])
+      logger.info("Loaded custom strings")
+    } catch (error) {
+      logger.error("Failed to load custom strings", field("error", error))
+      throw error
+    }
+  }
```

Comment:
We already add `Failed to load custom strings` to the error, and any errors thrown here will already be logged (I think?), so I am thinking we remove the catch to avoid logging the error twice.

```suggestion
  if (args["custom-strings"]) {
    await loadCustomStrings(args["custom-strings"])
    logger.info("Loaded custom strings")
  }
```